modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -28,6 +28,7 @@ import {
 	getComponentCache,
 	initComponentCache,
 } from './component.es';
+import delegate from './delegate/delegate.es';
```

Comment:
I think we should avoid providing this as a global since this would be a new delivery for `delegate`. See https://liferay.slack.com/archives/G3JBR21HA/p1604343092181200?thread_ts=1604053827.155100&cid=G3JBR21HA for more convo on that

Reply:
When I remove `delegate` from this file I can't see `delegate` in the `Liferay.Util` object, I think that's the point but I'm not sure how I can make sure JSPs can access it in this case. I'm not 100% sure about how all this hooks up without the import and export in `global.es.js` @wincent got any thoughts?

Reply:
Hey @kresimir-coko, that's the point, it wasn't there before, no reason it should be there now.

Before, in all cases, you were using it like `require="metal-dom/src/...."` so now you should be able to do something like `require="frontend-js-web"` and then get the `delegate` method from there.

Reply:
This is technically also wasteful since we're downloading most of the code twice in `global.es.js` and through the `main` entry in `frontend-js-web`,  but that's something we'll need to figure out later at a lower level.

Reply:
Okay, that makes sense, the `require` part was missing for me, I don't remember seeing `require="frontend-js-web"` elsewhere, but I guess that's the goal, we want to make it the go-to way of importing our global `Liferay.Util` object.